docs/conf.py

```diff
@@ -64,8 +64,10 @@
 autodoc_class_signature = "separated"
 autodoc_default_options = {
     "members": True,
+    "inherited-members": True,
     "member-order": "bysource",
     "undoc-members": True,
+    "exclude-members": "SchemaConfig,__init__,Config,attrs,chunk,copy,json,log",
```

Comment:
Need to try a few more approaches to remove the dunder methods too so that the inherited members are not recognised.

Am afraid am unable to do these changes until later this afternoon as have been caught up this start of week.

Reply:
yea no rush! thanks